jumanji/environments/routing/cleaner/env_test.py

```diff
@@ -39,17 +39,24 @@
 
 class DummyGenerator(Generator):
     def __init__(self) -> None:
-        super(DummyGenerator, self).__init__(num_rows=5, num_cols=5)
-
-    def __call__(self, key: chex.PRNGKey) -> Maze:
-        return SAMPLE_GRID
+        super(DummyGenerator, self).__init__(num_rows=5, num_cols=5, num_agents=N_AGENT)
```

Comment:
Could we state in the class docstring that we are creating a generator of size 5x5 with 3 agents and remove the constant `N_AGENT = 3`?

Reply:
Sure